src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1076,6 +1077,13 @@ internal async Task Cleanup(bool async, bool connectionClosing = false, bool isD
         /// <returns>The value of the specified column.</returns>
         public override decimal GetDecimal(int ordinal) => GetFieldValue<decimal>(ordinal);
 
+        /// <summary>
+        /// Gets the value of the specified column as a <see cref="SqlDecimal"/> object.
+        /// </summary>
+        /// <param name="ordinal">The zero-based column ordinal.</param>
+        /// <returns>The value of the specified column.</returns>
+        public SqlDecimal GetSqlDecimal(int ordinal) => GetFieldValue<SqlDecimal>(ordinal);
```

Comment:
We don't really add accessors on NpgsqlDataReader because it's kinda useless - users can just do `GetFieldValue<SqlDecimal>` directly... can you please remove this?